x-pack/plugins/cases/public/components/case_view/case_view_page.test.tsx

```diff
@@ -309,16 +321,25 @@ describe('CaseViewPage', () => {
         }}
       />
     );
-    userEvent.click(result.getByTestId('connector-edit').querySelector('button')!);
-    userEvent.click(result.getByTestId('dropdown-connectors'));
+
+    userEvent.click(screen.getByTestId('connector-edit').querySelector('button')!, undefined, {
+      skipPointerEventsCheck: true,
+    });
+    userEvent.click(screen.getByTestId('dropdown-connectors'), undefined, {
+      skipPointerEventsCheck: true,
+    });
+
     await waitForEuiPopoverOpen();
-    userEvent.click(result.getByTestId('dropdown-connector-resilient-2'));
 
-    await waitFor(() => {
-      expect(result.getByTestId('connector-fields-resilient')).toBeInTheDocument();
+    userEvent.click(screen.getByTestId('dropdown-connector-resilient-2'), undefined, {
+      skipPointerEventsCheck: true,
     });
 
-    userEvent.click(result.getByTestId('edit-connectors-submit'));
+    expect(await screen.findByTestId('connector-fields-resilient')).toBeInTheDocument();
```

Comment:
`findByTestId` does a `waitFor` internally so this may be a performance improvement 🤷‍♂️ maybe more a readability improvement.

Reply:
Reference: https://jeremywsherman.com/blog/2021/02/04/improving-a-testing-library-test/